styles/style.css

```diff
@@ -723,15 +706,374 @@
+    left: -180px;
+    left: -20%;
+    top: 100px;
+    background-image: url(../images/illustration-about-me-2-desktop.png);
+    background-size: contain;
+    background-repeat: no-repeat;
+  }
+
+  .about_me h1 {
+    height: fit-content;
+    width: 85%;
+    font-family: 'Inter', 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+    font-style: normal;
+    font-weight: bolder;
+    font-size: 45px;
+    line-height: 60px;
+    text-align: center;
+    color: #172b4d;
+    margin-bottom: 12px;
+  }
+
+  /* Grid container */
+  .about_me .about {
+    margin-top: 0;
+    margin-bottom: 30px;
+    position: relative;
+    width: calc(55% + 250px);
+    height: 316px;
+    margin-left: calc((100% - (55% + 250px)) / 2);
+    grid-gap: 8px;
+    grid-template-columns: 1fr 1fr 1fr;
+    justify-content: center;
+  }
+
+  .about .grid_2-item {
+    width: 100%;
+    margin-left: 0;
+    height: 100%;
+  }
+
+  /* ============================================================================================================================= */
+
+  /* Form section styling */
+  .form_s {
+    /* background: red; */
+    height: calc(16vw + 200px);
+    width: calc(55vw + 250px);
+    margin-left: calc((100vw - (55vw + 250px)) / 2);
+    top: 0;
+
+    /* content */
+    display: flex;
+    gap: 10px;
+    flex-direction: row;
+    padding: 10px 2px;
+    align-items: center;
+  }
+
+  .form_s h1 {
+    width: 45%;
+    margin-bottom: 0;
+
+    /* font-size: calc(30px); */
+    font-size: calc(1vw + 10px);
+    line-height: 52px;
+  }
+
+  .form_s .form {
+    width: 55%;
+  }
+
+  .form_s .form .submit {
+    justify-content: flex-start;
+  }
+
+  /* ===================================================================================================== */
+
+  /* Footer styling */
+  .img_bg1,
+  .img_bg2,
+  .img_bg3 {
+    /* width: 100%; */
+
+    /* background: green; */
+    background-image: url(../images/contact\ form-illustratoin-3-desktop.png);
+    background-size: cover;
+    background-repeat: no-repeat;
+    display: flex;
+    justify-content: center;
+    position: absolute;
+
+    /* width: 200px;
+    height: 450px; */
+    width: calc(6vw + 100px);
+    height: calc(14vw + 250px);
+    right: 0;
+    top: 2800px;
+
+    /* Betwenn screem  */
+
+    /* top: calc(215vw + 500px); */
+
+    /* bottom: 0; */
+    float: bottom;
+    z-index: 2;
+  }
+
+  .img_bg2 {
+    left: 0;
+    background-image: url(../images/illustration-1-contact-form-desktop.png);
+
+    /* width: 180px; */
+    width: calc(4vw + 100px);
+
+    /* height: 340px; */
+    height: calc(12vw + 150px);
+    top: 2925px;
+
+    /* top: calc(18vw + 250px); */
+    background-size: cover;
+  }
+
+  .img_bg3 {
+    /* left: 300px; */
+    left: 30vw;
+    background-image: url(../images/illustration-2-contact-form-desktop.png);
+
+    /* width: 115px; */
+    width: calc(4vw + 80px);
+
+    /* height: 90px; */
+    height: calc(1vw + 90px);
+    top: 3060px;
```

Comment:
- [ ] kindly consider removing each and every commented piece of unused code cause they make your code base look messy and make sure you don't leave any behind before you go for another review to make your codes professional

Reply:
Ok done